### ISSUE TITLE
Analytics code prevents building after cloning

### ISSUE THREAD

**Issue by SpacyRicochet** [vermont42/Conjugar#6]
After cloning the project, it's not possible to build it yet. The analytics code causes compiler errors, because the AWS frameworks are not included.

It's probably better to add some kind of dependency management so new contributors can more easily contribute.

*Workaround:* 

* Remove AWSFrameworks from the 'Embed Frameworks' build phase.
* Comment out script in the the Pinpoint Hocus Pocus build phase.
* Remove `awsconfiguration.json` from being copied in the Copy Resources build phase.
* Comment out `import AWSPinpoint` and all the contents of the methods in AWSAnalyticsService.swift.

**Comment by vermont42** [comment]
I agree that dependency management is the correct solution. In my defense, the app has had no external contributors for the three years of its existence, though there may soon be one. 😅 

As a stopgap, would you like to PR the readme with the workaround?

**Comment by SpacyRicochet** [comment]
Sure, that makes sense. I didn't expect this to be a very contributed to project either ;) But it's a fun one to learn some new accessibility features on!
